tools/nni_cmd/launcher.py

```diff
@@ -272,6 +274,17 @@ def set_experiment(experiment_config, mode, port, config_file_name):
 def launch_experiment(args, experiment_config, mode, config_file_name, experiment_id=None):
     '''follow steps to start rest server and start experiment'''
     nni_config = Config(config_file_name)
+
+    # check packages for tuner
+    if experiment_config.get('tuner') and experiment_config['tuner'].get('builtinTunerName'):
+        tuner_name = experiment_config['tuner']['builtinTunerName']
+        module_name = ModuleName[tuner_name]
+        try:
+            check_call([sys.executable, '-c', 'import %s'%(module_name)])
+        except ModuleNotFoundError as e:
+            print_error('The tuner %s should be installed through nnictl'%(tuner_name))
+            exit(1)
```

Comment:
@chicm-ms do you have any concern here?

Reply:
@xuehui1991 Do you mean the exit(1)?  Since here is in launcher, the rest server won't start if the checking fails, it won't be a big problem for code coverage tool here.  And there are already a couple of exit calls in launcher.